src/docker_bin/wo_docker_run

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+set -e
+
+## Argument checks.
+for arg in "$@"; do
+    if [ "$arg" = "--detach" ] || [ "$arg" = "-d" ]; then
+        echo "Should not give '--detach' argument to $0" >&2
+        exit 1
+    fi
+    if [ "$arg" = "--rm" ]; then
```

Comment:
`--rm` has to be a user-specifiable option — we'd like to give users the option for containers sticking around after they exit for debugging purposes.